core/tests/karma.conf.js

```diff
@@ -117,7 +117,8 @@ module.exports = function(config) {
     ],
     ngHtml2JsPreprocessor: {
       moduleName: 'directiveTemplates',
-      prependPrefix: '/'
+      prependPrefix: '/',
```

Comment:
Could you maybe add some comments explaining what these two key-value pairs do?

Reply:
Done.